### DISCUSSION TITLE
Support more language

### DISCUSSION THREAD

**Discussion by maxrave-dev** [sigma67/ytmusicapi#419] [Q&A]
Hi @sigma67, I see YouTube Music supports many languages but why in this library support too little. I try to add some language code to Constant.py but don't work now.
Hope you add more languages support

**Comment by sigma67** [comment]
Why? Because I'm not getting paid to do this, so I chose to support only the most popular languages, others can be contributed by anyone using that language.

See here if you want to add more languages yourself: https://github.com/sigma67/ytmusicapi/tree/master/ytmusicapi/locales

A lot of endpoints actually work without proper translations, if you disable the Exception here: https://github.com/sigma67/ytmusicapi/blob/master/ytmusicapi/ytmusic.py#L106

**Reply by maxrave-dev** [reply]
Thanks for your reply, "disable the Exception" and "add language code" working now

**Reply by sigma67** [reply]
I am open to changing this Exception to a Warning. Feel free to submit a PR
